index.bs

```diff
@@ -483,11 +493,22 @@ To <dfn>obtain an event attribution source from a source</dfn> given an [=attrib
 1. Set |resultSource|'s [=attribution source/source identifier=] to a new unique opaque string.
 1. Set |resultSource|'s [=attribution source/source type=] to "<code>event</code>".
 1. Round |resultSource|'s [=attribution source/expiry=] away from zero to the nearest day (86400 seconds).
-1. Set |resultSource|'s [=attribution source/attribution mode=] to one of
-    "<code>truthfully</code>", "<code>never</code>", or "<code>falsely</code>", with
-    [=implementation-defined=] probability.
+1. Let |never| be a random boolean, with true having an [=implementation-defined=] probability.
```

Comment:
I don't think this is right. We want `event source falsely mode rate` to include the probability that `never` is selected. There should only be a single probability that the impression will be reported non-truthfully.

Reply:
Just checking my logic for this: If the source's `attribution mode` was chosen to be `never`, then a report would never have been generated for it. So given that the report exists (by virtue of the reporting origin receiving the report), then we know that the source's `attribution mode` wasn't never. So shouldn't the probability only be `truthfully` vs `falsely`?

Reply:
The easiest way for me to think about the flip mechanism is like this:

1. Flip a coin with heads probability `p`
2. If the coin is heads --> report the event truthfully
3. If the coin is tails --> pick a random output with uniform probability

The choice of `never` is included in (3).

The mechanism as described here is:
1. Flip a coin with heads probability `p`
2. If heads --> Set the output to `never`
3. Else
    1. Flip another coin with heads probability `q`
    2. If heads --> report truthfully
    3. Else pick a random output (other than `never`) with uniform probability

The thing that is confusing here is that the initial `p` probability is very constrained by 3.3 since we want `never` to be picked just as likely as any of the other output states.

Reply:
> If the coin is tails --> pick a random output with uniform probability

Is `truthfully` included in those potential outputs, or does it include only the two non-`truthfully` outputs of `never` and `falsely`?

I think it makes sense for me to reword the spec in a separate PR before continuing with this one.